third_party/proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -85,8 +85,8 @@ message Validator {
   option (gogoproto.goproto_stringer) = false;
   option (gogoproto.goproto_getters)  = false;
 
-  // operator_address defines the address of the validator's operator; bech encoded in JSON.
-  string              operator_address = 1 [(gogoproto.moretags) = "yaml:\"operator_address\""];
+  // oracle_address defines the address of the validator's operator; bech encoded in JSON.
+  string              oracle_address = 1 [(gogoproto.moretags) = "yaml:\"oracle_address\""];
```

Comment:
Oh, `staking.proto` has also been changed. Please check again :)

Reply:
Oh. I missed it. 
Thank you.